index.js

```diff
@@ -369,7 +369,7 @@ async function handleSensorRequest(currentLocationId, radarType) {
             fallbackTimeoutMillis: location.fallbackTimer,
             reminderMessage: `This is a reminder to check on the bathroom`,
             fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
-            fallbackToPhoneNumber: location.fallbackNumbers,
+            fallbackToPhoneNumbers: location.fallbackNumbers,
```

Comment:
This is probably a bug in what is in `devenv` right now. So this will need to be merged before we push that live. In fact, I'll just make a new PR for this one-liner to make sure that it makes it in.

Reply:
https://github.com/bravetechnologycoop/BraveSensor-Server/pull/99